kubernetes_deploy/production/ingress.yml

```diff
@@ -15,3 +19,10 @@ spec:
         backend:
           serviceName: laa-cla-public
           servicePort: 80
+  - host: checklegalaid.service.gov.uk
+    http:
```

Comment:
_not a blocker_
Is there a way to re-use the previous definition? With alternative syntax or YAML anchors?